libcaf_net/src/tcp_accept_socket.cpp

```diff
@@ -118,7 +118,7 @@ make_tcp_accept_socket(const uri::authority_type& node, bool reuse_addr) {
   if (auto ip = get_if<ip_address>(&node.host))
     return make_tcp_accept_socket(ip_endpoint{*ip, node.port}, reuse_addr);
   auto host = get<std::string>(node.host);
-  auto addrs = ip::resolve(host);
+  auto addrs = ip::resolve(host); // TODO: switch to local_addresses()
```

Comment:
Why add a TODO instead of doing the switch?

Reply:
I forgot about that.